kotest-property/src/jvmMain/kotlin/io/kotest/property/seed/seedio.kt

```diff
@@ -22,9 +22,15 @@ actual fun readSeed(path: TestPath): Long? {
 fun seedDirectory(): Path = Paths.get(System.getProperty("user.home")).resolve(".kotest").resolve("seeds")
 
 fun seedPath(path: TestPath): Path {
-   return seedDirectory().resolve(path.value)
+   return seedDirectory().resolve(escape(path.value))
 }
 
+fun escape(path: String) = path
```

Comment:
could make this `private`, or `internal`?

```suggestion
private fun escape(path: String) = path
```